lib/improver/blending/blend_across_adjacent_points.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
```

Comment:
Yep, I think we might want to use slightly different weighting functions in here, so I left it open for that.